pages/blog/agriwebinar.md

```diff
@@ -1,10 +1,10 @@
 ---
 date: 2025-04-17
 title: April Agriculture Science Cluster Webinar
-image: https://esa-earthcode.github.io/portal-assets/blog/Living_Planet_Symposium_2025_pillars-4170729331.jpg
+image: https://raw.githubusercontent.com/ESA-EarthCODE/portal-assets/8ceea8ea71c8834a510c2311a80c9f40c55f49cf/blog/agriculture_header2.jpg
```

Comment:
Please use the link to the GH pages deployment instead: https://esa-earthcode.github.io/portal-assets/blog/agriculture_header2.jpg